[2022-06-22 08:20:06,341] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: DM-COUNT-account-turnover.START_COUNT_BY_DAY_8 manual__2022-06-22T08:19:49.731523+00:00 [queued]>
[2022-06-22 08:20:06,458] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: DM-COUNT-account-turnover.START_COUNT_BY_DAY_8 manual__2022-06-22T08:19:49.731523+00:00 [queued]>
[2022-06-22 08:20:06,458] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-06-22 08:20:06,458] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-06-22 08:20:06,459] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-06-22 08:20:06,521] {taskinstance.py:1259} INFO - Executing <Task(PostgresOperator): START_COUNT_BY_DAY_8> on 2022-06-22 08:19:49.731523+00:00
[2022-06-22 08:20:06,578] {standard_task_runner.py:52} INFO - Started process 1944 to run task
[2022-06-22 08:20:06,589] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'DM-COUNT-account-turnover', 'START_COUNT_BY_DAY_8', 'manual__2022-06-22T08:19:49.731523+00:00', '--job-id', '983', '--raw', '--subdir', 'DAGS_FOLDER/dm_fill_account_turnover.py', '--cfg-path', '/tmp/tmpp6aorblc', '--error-file', '/tmp/tmpqgt9olma']
[2022-06-22 08:20:06,590] {standard_task_runner.py:77} INFO - Job 983: Subtask START_COUNT_BY_DAY_8
[2022-06-22 08:20:06,845] {logging_mixin.py:109} INFO - Running <TaskInstance: DM-COUNT-account-turnover.START_COUNT_BY_DAY_8 manual__2022-06-22T08:19:49.731523+00:00 [running]> on host a0a302277daf
[2022-06-22 08:20:07,359] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=DM-COUNT-account-turnover
AIRFLOW_CTX_TASK_ID=START_COUNT_BY_DAY_8
AIRFLOW_CTX_EXECUTION_DATE=2022-06-22T08:19:49.731523+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-22T08:19:49.731523+00:00
[2022-06-22 08:20:07,380] {base.py:79} INFO - Using connection to: id: postgres. Host: postgres, Port: 5432, Schema: postgres, Login: ***, Password: ***, extra: {}
[2022-06-22 08:20:07,390] {dbapi.py:225} INFO - Running statement: CALL ds.fill_account_turnover_f (%(date)s::date);, parameters: {'date': '2018-01-8'}
[2022-06-22 08:20:07,564] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=DM-COUNT-account-turnover, task_id=START_COUNT_BY_DAY_8, execution_date=20220622T081949, start_date=20220622T082006, end_date=20220622T082007
[2022-06-22 08:20:07,649] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-06-22 08:20:07,733] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
